Wikipedia/Code/CollectionViewEditControllerNavigationDelegate+Extensions.swift

```diff
@@ -20,21 +20,21 @@ extension CollectionViewEditControllerNavigationDelegate where Self: UIViewContr
                 batchEditToolbarViewController.view.alpha = 0
             }
         }
-        
-        batchEditToolbarViewController.apply(theme: currentTheme)
-        UIView.animate(withDuration: 0.3, delay: 0, options: [.beginFromCurrentState, .curveLinear], animations: {
+
+        UIView.transition(with: batchEditToolbarViewController.view, duration: 0.7, options: .transitionCrossDissolve, animations: {
             if let tabBar = tabBar {
                 tabBar.alpha = isHidden ? 1 : 0
             } else {
                 batchEditToolbarViewController.view.alpha = isHidden ? 0 : 1
```

Comment:
By using `UIView.transition` without `UIViewAnimationOptionAllowAnimatedContent`, I don't think these changes have any effect. You might be able to remove them entirely and just add and remove the subview.

https://developer.apple.com/documentation/uikit/uiview/1622574-transitionwithview